### PR TITLE
[BISERVER-12825] - The width of the field which displays the roles in

### PR DESCRIPTION
**Pull request by stanislau-strelchanka**

### CODE REVIEW

**Comment by stanislau-strelchanka** [comment]
@mdamour1976, @rmansoor, @e-cuellar, @mdamour1976, @pentaho-nbaker, @mchen-len-son, @pamval please review

**Comment by pamval** [comment]
@stanislau-strelchanka can you please provide a simple explanation on how this fixes the issue?

thanks!

**Comment by stanislau-strelchanka** [comment]
@pamval sure, it sets the title attribute of the option element so that there will be a tooltip on mouse hover.

**Comment by stanislau-strelchanka** [comment]
@pamval also, I've spent some time investigating on how horizontal bar could be added to List Box widget, but looks like there is no simple way of doing it except creating custom widget with panels, labels, etc.

**Comment by stanislau-strelchanka** [comment]
@mdamour1976, @rmansoor, @e-cuellar, @mdamour1976, @pentaho-nbaker, @mchen-len-son, @pamval updated, please review

**Comment by stanislau-strelchanka** [comment]
@mdamour1976, @rmansoor, @e-cuellar, @mdamour1976, @pentaho-nbaker, @mchen-len-son, @pamval please review

**Comment by pamval** [comment]
@stanislau-strelchanka please squash these commits. thanks!

**Comment by stanislau-strelchanka** [comment]
@pamval updated pr

**Comment by pamval** [comment]
@rmansoor, @e-cuellar, @mdamour1976, @pentaho-nbaker, @mchen-len-son, please review

**Comment by stanislau-strelchanka** [comment]
@mdamour1976, @rmansoor, @e-cuellar, @mdamour1976, @pentaho-nbaker, @mchen-len-son, @pamval please review

File: user-console/source/org/pentaho/mantle/client/ui/custom/HorizontalScrollWrapper.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * Class that wraps list box into div necessary for horizontal scroll effect
+ */
+public class HorizontalScrollWrapper extends Composite {
+
+  private String id;
+
+  public HorizontalScrollWrapper( ListBox listBox ) {
+    id = getListBoxWrapperUIId( listBox );
+    String html = ""
+      + "<div id='" + id + "' style=\"overflow-x:scroll; width:200px; overflow: -moz-scrollbars-horizontal;\">"
```
Comment (mbatchelor) [inline user-console/source/org/pentaho/mantle/client/ui/custom/HorizontalScrollWrapper.java]: I believe this is really bad form - embedded style/html in java code. Shouldn't this be in a CSS file ?

**Pull request merged**